lib/house.rb

```diff
@@ -0,0 +1,62 @@
+
+require './lib/room'
+
+class House
+  attr_reader :price,
+              :address
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @area = area
```

Comment:
This seems like dead code. We don't set this as `attr_reader` we already have and `#area` method here in the class, and we don't reference the instance variable here in the class